src/components/UniversalResults.tsx

```diff
@@ -10,12 +14,13 @@ import { VerticalConfigMap } from '../models/verticalConfig';
  *
  * @public
  */
-export interface UniversalResultsCssClasses {
+export interface UniversalResultsCssClasses extends SectionHeaderCssClasses {
   container?: string,
   results___loading?: string
 }
 
 const builtInCssClasses: UniversalResultsCssClasses = {
+  ...sectionHeaderCssClasses,
```

Comment:
Can we be consistent about whether we spread the css classes or specify them individually?

Reply:
Sure, I updated to specify them individually